lxc/main.go

```diff
@@ -333,7 +333,13 @@ func (c *cmdGlobal) PreRun(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	} else {
-		c.conf = config.NewConfig(filepath.Dir(c.confPath), true)
+		confDir := filepath.Dir(c.confPath)
+		if !shared.PathExists(confDir) {
+			if err = os.MkdirAll(confDir, 0700); err != nil {
```

Comment:
LXD doesn't use the single-line if syntax in our codebase.
Please separate the assignment and check.